.evergreen/.evg.yml

```diff
@@ -353,6 +353,7 @@ functions:
         script: |
           ${PREPARE_SHELL}
           cd ${DRIVERS_TOOLS}/.evergreen/auth_aws
+          . ./activate_venv.sh
```

Comment:
Why is this now required?

Reply:
I don't know, actually.  I copied it from pymongo as per @ShaneHarvey 's suggestion.  @bazile-clyde , do you know?

Reply:
It's because of [DRIVERS-1462](https://jira.mongodb.org/browse/DRIVERS-1462). The auth testing scripts (written in python) depend on the boto3 library. The linked issue was caused by the Ubuntu 18.04 evergreen host using an old version of boto3. Rather than having every driver upgrade boto3 themselves (or potentially any other dependencies), `activate_venv.sh` abstracted this complexity away. The script sets up a virtual environment and upgrades boto3.

Reply:
Hopefully, if we run into this same type of issue we can upgrade `activate_venv.sh` without causing driver-wide changes.

Reply:
Sounds good thanks @bazile-clyde